apps/i18n/common/en_us.json

```diff
@@ -1935,6 +1935,7 @@
   "resetPassword": "Reset password",
   "resetTeacherPasswordTooltip": "You cannot reset passwords for teacher accounts in your section.",
   "secretLoginTooltip": "Disabled for teacher accounts in your section",
+  "familyNameTooltip": "Disabled for teacher accounts in your section",
```

Comment:
a) Do we typically create a new string for each usage instead of reusing them when the text is identical? In my previous company the advice was to reuse them when possible to minimize the amount of data going to the client, but I could see benefits either way (easier to change text when you know it's only going to affect the one spot).
b) If we are to use just one string it would be ideal to give it a key not specific to the original location where it was used, but if changing the string's key messes with the translation process I'm fine leaving it as is.
c) Seems like 99% of the strings in this file are ordered alphabetically by their keys. I see a very small number of counterexamples (including `secretLoginTooltip` just above), but let's not add more.

Reply:
Good point - I've made it a more universal key and updated where it's used in both places. Once a full sync occurs (and I have to wait for this in order for the new string to inherit the existing translation in Crowdin), I will be able to remove the old `secretLoginTooltip` string as a follow-up task.